src/settings.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Deserialize, Serialize};
+
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct Settings {
+    pub(crate) denied_storage_classes: Vec<String>,
```

Comment:
You should use a HashSet instead, it's better in this case. It removes duplicates and provides faster lookup